plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -880,7 +884,12 @@ private ConnectorTableMetadata getTableMetadata(ConnectorSession session, Schema
             properties.put(LOCATION_PROPERTY, icebergTable.location());
         }
 
-        return new ConnectorTableMetadata(table, columns, properties.buildOrThrow(), getTableComment(icebergTable));
+        IcebergUtil.getSplitSize(icebergTable).map(value -> properties.put(SPLIT_SIZE_PROPERTY, value));
```

Comment:
Similar to the comment is `IcebergTableProperties`, I don't think we need to do this here.